arcgis-ios-sdk-samples/Maps/Show location history/README.metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+    "category": "Maps",
+    "description": "Display your location history on the map.",
+    "ignore": false,
+    "images": [
+        "image1.png"
+    ],
+    "keywords": [
+		"GPS", 
```

Comment:
Please use four spaces rather than tabs for indentation, to match the rest of the file.

Reply:
Apologies for overlooking this in my text editor setup. I tidied this up in [`2e108ddc`](https://github.com/Esri/arcgis-runtime-samples-ios/pull/751/commits/2e108ddc08e24501431b9b8cefae28dfdc4e5c5d).